packages/arb-token-bridge-ui/src/components/common/SiteBanner.tsx

```diff
@@ -1,39 +1,57 @@
-import dayjs from 'dayjs'
 import { useEffect, useState } from 'react'
+import dayjs from 'dayjs'
+import utc from 'dayjs/plugin/utc'
 import { ExternalLink } from './ExternalLink'
 import { ArbitrumStatusResponse } from '../../pages/api/status'
 import { getAPIBaseUrl } from '../../util'
 
-const SiteBannerArbiscanIncident = () => {
+dayjs.extend(utc)
+
+const SiteBannerArbiscanIncident = ({
+  type
+}: {
+  type: 'arbitrum-one' | 'arbitrum-nova'
+}) => {
+  const explorerUrl =
+    type === 'arbitrum-nova'
+      ? 'https://nova.arbiscan.io/'
+      : 'https://arbiscan.io/'
+  const explorerTitle = type === 'arbitrum-nova' ? 'Nova Arbiscan' : 'Arbiscan'
+  const alternativeExplorerUrl =
+    type === 'arbitrum-nova' ? '' : 'https://www.oklink.com/arbitrum'
+
   return (
     <div className="bg-orange-dark px-4 py-[8px] text-center text-sm font-normal text-white">
       <div className="w-full">
         <p>
-          <ExternalLink className="underline" href="https://arbiscan.io/">
-            Arbiscan
+          <ExternalLink className="underline" href={explorerUrl}>
+            {explorerTitle}
           </ExternalLink>{' '}
           is temporarily facing some issues while showing the latest data.
-          Arbitrum chains are still live and running. If you need an alternative
-          block explorer, you can visit{' '}
-          <ExternalLink
-            className="underline"
-            href="https://www.oklink.com/arbitrum"
-          >
-            OKLink
-          </ExternalLink>
-          .
+          Arbitrum chains are still live and running.
```

Comment:
in terms of the copy, if we are doing chain-specific banners, it should specify the chain name here, not `Arbitrum chains`

Reply:
Updated